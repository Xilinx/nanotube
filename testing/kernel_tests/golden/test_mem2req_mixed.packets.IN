###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
2020-03-26 18:44:17.184404
0000  a0 f3 c1 8c 29 58 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2a af 00 00 80 11 8d e8 c0 a8 00 e7 c0 a8
0020  00 01 da a0 1a 85 00 09 88 7d 00
2020-03-26 18:45:34.076120
0000  a0 f3 c1 8c 29 58 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2c 3b 00 00 80 11 8c 5c c0 a8 00 e7 c0 a8
0020  00 01 e5 d5 1a 85 00 09 7c 48 01
2020-03-26 18:45:47.762601
0000  a0 f3 c1 8c 29 58 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2c 81 00 00 80 11 8c 16 c0 a8 00 e7 c0 a8
0020  00 01 d3 b1 1a 85 00 09 8d 6c 02
2020-03-26 18:45:58.843210
0000  a0 f3 c1 8c 29 58 58 a0 23 18 a6 15 08 00 45 00
0010  00 1d 2c c4 00 00 80 11 8b d3 c0 a8 00 e7 c0 a8
0020  00 01 ef f6 1a 85 00 09 70 27 03
