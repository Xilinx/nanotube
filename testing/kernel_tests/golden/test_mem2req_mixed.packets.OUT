###########################################################################
# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
# SPDX-License-Identifier: MIT
###########################################################################
1970-01-01 00:00:00.000000
0000  11 22 33 44 55 66 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2a af 00 00 80 11 8d e8 c0 a8 00 e7 c0 a8
0020  00 01 da a0 1a 85 00 09 88 7d 00
1970-01-01 00:00:00.000000
0000  11 22 33 44 55 66 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2c 3b 00 00 80 11 8c 5c c0 a8 00 e7 c0 a8
0020  00 01 e5 d5 1a 85 00 09 7c 48 01
1970-01-01 00:00:00.000000
0000  11 22 33 44 55 66 58 a0 23 18 a6 14 08 00 45 00
0010  00 1d 2c 81 00 00 80 11 8c 16 c0 a8 00 e7 c0 a8
0020  00 01 d3 b1 1a 85 00 09 8d 6c 02
